plugins.json

```diff
@@ -628,6 +611,23 @@
         "title": "Database Storage",
         "version": "1.0.2"
     },
+    "DefinitionOfDone": {
+        "author": "Tomas Dittmann",
+        "compatible_version": ">=1.2.32",
+        "description": "This plugin adds a simple checklist to the taskview, similar to subtasks.<br>Each entry consists of a title-, a description- and a toggleable \"done\"-field.<br><span style=\"color: #0044ff; font-size: 1.2em;\">&nbsp;&#x26A0;&nbsp;</span>Compatible with [MarkdownPlus](https://github.com/creecros/MarkdownPlus/releases)",
```

Comment:
oh I copy-pasted it from the CCIT plugin definition.
is the description parsed with markdown?
if so - markdownplus takes care of inline-html for me, but I'd still change it to simple markdown syntax

Reply:
@fguillot thanks for that, makes more sense with your explanation.

Reply:
@fguillot This will work. No html, tested with `MARKDOWN_ESCAPE_HTML` option to `true` (which is the default)

![image](https://github.com/kanboard/website/assets/10233708/e67d0f29-511c-47ab-8314-84c76f68e534)
---
@Chaosmeister I suggest you change your description to something like:
```json
"description": "This plugin adds a simple checklist to the taskview, similar to subtasks. Each entry consists of a title, a description and a toggleable \"**done**\" field. \n &nbsp;⚠️&nbsp; Compatible with [MarkdownPlus](https://github.com/creecros/MarkdownPlus/releases)",
```
**which produces:**
![image](https://github.com/kanboard/website/assets/10233708/d48abdb4-8b7d-476f-9e58-b97e89d1775c)
---
@fguillot Once this commit is resolved or merged, I will fix all the (few) descriptions which contain the inline html. If you merge this commit as it is, I will include it in the fix tonight.

Reply:
awesome,  thanks alot!